KGP.TicketApp.Contracts/IRepositoryBase.cs

```diff
@@ -0,0 +1,13 @@
+﻿using System.Linq.Expressions;
+
+namespace KGP.TicketApp.Contracts
+{
+    public interface IRepositoryBase<T>
+    {
+        IQueryable<T> FindAll();
+        IQueryable<T> FindByCondition(Expression<Func<T, bool>> expression);
+        void Create(T entity);
+        void Update(T entity);
```

Comment:
a Update i delete mogłyby zwracać boola czy się udało. Chyba że obsługujemy wyjątki gdzieś indziej, może do tego wrócę i zobaczę

Reply:
Jak entity coś takiego udostepnia a pewnie tak to dodam

Reply:
zawsze można rzucać wyjątki. Tylko wtedy przyda się komentarz:
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/language-specification/documentation-comments#d35-exception